pkg/task/storage.go

```diff
@@ -169,16 +170,19 @@ func (s *Storage) changePrefix(dst string, src string, id string) error {
 	}
 	val, err := trans.Get(oldkey, nil)
 	if err != nil {
+		logging.S().Errorf("error when trying to get the old key: '%s', err: %v", oldkey, err)
 		trans.Discard()
 		return err
 	}
 	err = trans.Put(newkey, val, &opt.WriteOptions{Sync: true})
 	if err != nil {
+		logging.S().Errorf("error when trying to put the new key: '%s', err: %v", newkey, err)
 		trans.Discard()
 		return err
 	}
 	err = trans.Delete(oldkey, &opt.WriteOptions{Sync: true})
 	if err != nil {
+		logging.S().Errorf("error when trying to drop the old key: '%s', err: %v", oldkey, err)
 		trans.Discard()
 		return err
 	}
```

Comment:
Let's drop this, that's code from https://github.com/testground/testground/issues/1418#issuecomment-1203819103